Client/Frontend/AuthenticationManager/RemovePasscodeViewController.swift

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import SwiftKeychainWrapper
+import Shared
+
+let NotificationPasscodeDidRemove   = "NotificationPasscodeDidRemove"
+
+/// Displayed to the user when removing a passcode.
+class RemovePasscodeViewController: PagingPasscodeViewController, PasscodeInputViewDelegate {
+    private var confirmCode: String?
+
+    override init() {
+        super.init()
+        self.title = AuthenticationStrings.turnOffPasscode
+        self.panes = [
+            PasscodePane(title: AuthenticationStrings.enterPasscode),
+            PasscodePane(title: AuthenticationStrings.reenterPasscode),
```

Comment:
No point in carrying this over -- might as well fix [bug 1252070](https://bugzilla.mozilla.org/show_bug.cgi?id=1252070) while here by not including the second pane.